packages/shared/graphql/generated/GetRecording.ts

```diff
@@ -150,6 +150,8 @@ export interface GetRecording_recording {
   operations: any | null;
   resolution: any | null;
   metadata: any | null;
+  isTest: boolean;
```

Comment:
I think the only thing the frontend should be using this new attribute to do is e.g. show the Test Suites side panel.